src/index.html

```diff
@@ -1,22 +1,390 @@
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      class="category__photo"
+                      src="./images/categories/img-5.png"
+                      alt="Smart Home"
+                    />
+                  </a>
+
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      class="category__photo"
+                      src="./images/categories/img-6.png"
+                      alt="Smart Home"
+                    />
+                  </a>
+                </div>
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+              <div class="categories__button-wrapper">
+                <button class="button categories__button">View all</button>
+              </div>
+            </div>
+          </section>
+        </div>
+      </div>
+      <section
+        class="about"
+        id="about"
+      >
+        <div class="about__photo"></div>
+        <div class="about__content">
+          <h2 class="section-title about__title">
+            Timeless, for 50 years and counting
+          </h2>
+          <p class="about__description">
+            Bang & Olufsen is expanding its Recreated Classics Program with the
+            launch of Beosystems – a limited edition music system that
+            transcends time by bridging the gap between one of our iconic
+            designs from 1972 and today’s cutting-edge digital technology.
+          </p>
+        </div>
+      </section>
+      <div class="container">
+        <div class="main__content">
+          <section
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__wrapper">
+              <form
+                class="contacts__form"
+                onsubmit="this.reset(); return false;"
+              >
+                <input
+                  class="contacts__input"
+                  type="text"
+                  name="name"
+                  required
+                  placeholder="Name"
+                />
+                <input
+                  class="contacts__input"
+                  type="email"
+                  name="email"
+                  required
+                  placeholder="Email"
+                />
+                <textarea
+                  class="contacts__input contacts__input--message"
+                  name="message"
+                  placeholder="Message"
+                  required
+                ></textarea>
+                <button class="button contacts__button">Send</button>
+              </form>
+
+              <div class="contacts__info">
+                <div class="contacts__info--block">
```

Comment:
I believe you wanted an *Element* called *info-block* (with only 1 hyphen). Two hyphens means it's a modifier.
In this case, I don't see `block`, `title` and `text` as being modifiers, but part of the element name